src/EnergyPlus/FanCoilUnits.hh

```diff
@@ -349,7 +349,8 @@ namespace FanCoilUnits {
                           int const ControlledZoneNum,   // ZoneEquipConfig index
                           bool const FirstHVACIteration, // flag for 1st HVAV iteration in the time step
                           Real64 &LoadMet,               // load met by unit (watts)
-                          Optional<Real64> PLR = _       // Part Load Ratio, fraction of time step fancoil is on
+                          Optional<Real64> PLR = _,      // Part Load Ratio, fraction of time step fancoil is on
+                          Real64 ElecHeatCoilPLR = 1.0   // electric heating coil PLR used with MultiSpeedFan capacity control 
```

Comment:
The new argument *ElecHeatCoilPLR* is defaulted to 1.0 instead of making it optional argument. However, I may have to rename the argument to reflect that it is cycling ratio, instead of PLR.